quipucords/quipucords/settings.py

```diff
@@ -112,9 +112,13 @@
 
 WSGI_APPLICATION = 'quipucords.wsgi.application'
 
+DEFAULT_PAGINATION_CLASS = 'rest_framework.pagination.PageNumberPagination'
+
 REST_FRAMEWORK = {
     'DEFAULT_FILTER_BACKENDS':
         ('django_filters.rest_framework.DjangoFilterBackend',),
+    'DEFAULT_PAGINATION_CLASS': DEFAULT_PAGINATION_CLASS,
+    'PAGE_SIZE': 10
```

Comment:
Is this fixed or can it be specified per query?

Reply:
This is just the default. Can be overridden.